operator/webhook/test/test_sync.py

```diff
@@ -70,38 +90,64 @@ def test_spring_app_config_json_missing_props_and_secret_sources(full_route):
 
     assert spring_conf["name"] == "SPRING_APPLICATION_JSON"
 
-    expected_json = {"spring": {"application": {"name": "testroute"}}}
+    expected_json = {"spring": {"application": {"name": "testroute"}}, "management": {"server": {"port": 80, "ssl": {"enabled": False}}}}
     assert json_props == expected_json
 
 
-def test_jdk_options_pkcs12_type(full_route):
-    options = _get_java_jdk_options(full_route)
+def test_tls_options_pkcs12_type(full_route):
+    options = _get_tls_options(full_route)
 
-    assert options["name"] == JDK_OPTIONS_ENV_NAME
+    assert len(options) > 0
 
-    expected_options = "-Djavax.net.ssl.trustStore=/etc/cabundle/test-truststore.p12 -Djavax.net.ssl.trustStorePassword= -Djavax.net.ssl.trustStoreType=PKCS12"
-    assert options["value"] == expected_options
+    truststore_key = next((x for x in options if x.get('name') == 'SERVER_SSL_TRUSTSTORE'), None)
+    truststore_type = next((x for x in options if x.get('name') == 'SERVER_SSL_TRUSTSTORETYPE'), None)
```

Comment:
:memo: Alternative to using the generator, you could transform the env var list into a dictionary:
```
envs = {x['name']: x['value'] for x in options}
```
Then you can just check for the keys you are interested in with `.get`. Might make things a little easier to read by reducing duplication.